VoltAir/Engine/Engine.cpp

```diff
@@ -125,6 +125,7 @@ void Engine::init() {
     mDestructionListener.reset(new DestructionListener());
     mDebugDraw.reset(new LiquidFunDebugDraw());
 
+    mValidLetters = "AB";
```

Comment:
@diegowald Can this be stored an fetched later? For example, store when an update is received via the Broadcasted Intent, and then fetched the day after?